### ISSUE TITLE
Stack label overlaps with Input text field

### ISSUE THREAD

**Issue by svlada** [GeekyAnts/NativeBase#1533]
## react-native, react and native-base version
react: 16.0.0
react-native: 0.51.0
native-base: 2.3.5

## Expected behaviour
Stacked label should be above Input text field.

## Actual behaviour
Stack label overlapping with Input text field, preventing us to enter text in the input field

## Steps to reproduce (code snippet or screenshot)
```
  render() {
    return (
      <Container style={commonStyles.container}>
        <Header>
          <Left>
            <Button transparent onPress={() => this.props.navigation.navigate('DrawerOpen')}>
              <Icon name='menu' />
            </Button>
          </Left>
          <Body>
            <Title>Patient</Title>
          </Body>
          <Right />
        </Header>
        <View>
        <Item stackedLabel>
            <Label>
              {this.state.label}
            </Label>
          <Input  />
        </Item>
        </View>
      </Container>
    );
  }
```
## Screenshot of emulator/device
![photo_2018-01-21_19-46-31](https://user-images.githubusercontent.com/845095/35197735-371be00a-fee4-11e7-9990-462888712fc6.jpg)

## Is the bug present in both ios and android or in any one of them?
Both
## Any other additional info which would help us debug the issue quicker.
Stacked Label and Input text field are displayed correctly if wrapped with Content component.

**Comment by akhil-geekyants** [comment]
@svlada wrap it inside `<Content/>` component or `<ScrollView/>`

Gif

![stacked label](https://user-images.githubusercontent.com/27768184/35267353-00b538f4-004c-11e8-8c55-67a31c4b680a.gif)

**Comment by svlada** [comment]
@akhil-geekyants 

In the issue description I have already stated that label is displayed correctly if wrapped inside of Content. Input component shouldn't be dependent on wrapper component. 

I have a use case with **sticky search text field** positioned on top of the screen and result list that's shown below the text field. Problem is that you cannot have fixed/sticky View inside of Content component which is actually Scrollable View.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.3.9](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.3.9)
